ietf/person/forms.py

```diff
@@ -21,3 +21,19 @@ def get_person(self, pk):
             return Person.objects.get(pk=pk)
         except Person.DoesNotExist:
             raise forms.ValidationError("ID does not exist")
+
+
+class MergeRequestForm(forms.Form):
+    to = forms.CharField()
+    frm = forms.CharField()
+    reply_to = forms.CharField()
+    subject = forms.CharField()
+    body = forms.CharField(widget=forms.Textarea)
+
+    def clean_to(self):
+        addresses = self.cleaned_data['to']
+        return addresses.split(',')
```

Comment:
Not ideal to be "cleaning" a CharField into a list in the `clean_*` method. That sort of transformation needs to be in a `to_python()` method of a custom field class. We have `ietf.utils.fields.MultiEmailField` that might be close to what you need, though it is probably in need of some clean up.

If that's not doing the right things, it'd be worth making your own field class here.

Reply:
Ok. I mixed this up. For the send_mail_text() "to" and "reply-to", which expect lists, the existing MultiEmailField works fine. For the From address, which gets a name-addr style email address from settings.DEFAULT_FROM_EMAIL, I created a custom field NameAddrEmailField.

See commit 421252641d2eab5837ed5e321f713b4e18969437